apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -625,6 +625,15 @@ export const database: NavMenuConstant = {
         { name: 'Managing secrets with Vault', url: '/guides/database/vault' },
       ],
     },
+    {
+      name: 'Connection management',
+      url: undefined,
+      items: [
+        { name: 'Supavisor', url: '/guides/database/supavisor' },
```

Comment:
I don't think we should use `Supavisor` in the sidebar. New developers don't even know what it is

IMO this entire page could be a section in the `Connection Pooling` page